tests/report_tests/plugin_tests/teamd.py

```diff
@@ -25,6 +25,9 @@ class TeamdPluginTest(StageTwoReportTest):
     sos_cmd = '-o teamd'
     redhat_only = True
 
+    # teaming has been deprecated from RHEL 9
+    versions = ['8']
```

Comment:
Ah I realize: `teamd` is also on RHEL7 so we shall allow tests of the plugin on version 7 also. But we (sos upstream) dont further support so old Python version that is on RHEL7, so..?

Reply:
It's tricky, isn't it? I was thinking of having a min version and max version and use ranges, and that could have covered that case. So it's not just a case of RHEL7, rather how to deal with possible ranges - what's the most elegant and pythonic way to do. If I say that this test runs in RHEL 8, do we assume that that's the last version where it ran?
And what about Ubuntu? How do we cover their cases?
It may be simpler to be explicit only when something has been deprecated, so for something that will get deprecated in RHEL10, we explicitly write '8', '9'.

Reply:
This was already passing here with centos Stream 9, or is the basis here that this is done in downstream testing?

Reply:
It's mainly to do with RHEL testing, but the way I implemented it should cover any other distro that have such issue

Reply:
This raises more generic concern that we can imho ignore now: how to deal with requirement "test this on RHEL/CentOS 8 *or* Ubuntu XY" - the `versions` would have to be a distro->versions mapping then.

But since we need this feature for one distro-family only, no need for that (now).

Reply:
Would this work?
	versions = ['8', '9', '22.04']

Or do we rather prefer something like:
	versions = ['rhel8', 'rhel9', 'ubuntu22.04']

Reply:
OK, my bad example since Ubuntu version cant be confused with RHEL version and most probably there isn't a pair of distros with similar version number that we would need to differentiate for the tests.

I would keep with the current "test version only (of either distro)" since that is sufficient for our current need. If we would require more in future, let open a new PR (that time).